cpg_infra/config/config.py

```diff
@@ -205,6 +213,13 @@ class GCPAggregator(DeserializableDataclass):
     # within itself, but this is a map of all users known to the system
     users: dict[MemberKey, CPGInfrastructureUser]
 
+    # include list of plugins enabled. This is specified explicitly to control
+    # what plugins are included from dependencies, also to allow exclusion of
+    # plugins for testing purposes
+    plugins_enabled: list[str] = dataclasses.field(
+        default_factory=lambda: ['metamist', 'billing_aggregator'],
```

Comment:
Happy for this to be always specific in the config rather than the two plugins being the default. I like this though, if a plugin isn't installed, it will fail.

Reply:
I made it the default mostly so that the linting on this branch would pass. I can definitely remove but we'll need to merge the private PR first or this one will fail when it tries to preview because of the missing config.

Reply:
I've removed the default value, we can just co-ordinate the merges to do the private one first